compiler/parser/expression_parsing.go

```diff
@@ -319,6 +319,26 @@ func (p *Parser) parseInfixExpression(left ast.Expression) ast.Expression {
 	return exp
 }
 
+func (p *Parser) parseAssignExpression(v ast.Expression) ast.Expression {
+	variable, ok := v.(ast.Variable)
+
+	if !ok {
+		p.error = &Error{Message: fmt.Sprintf("Can't assign value to %s. Line: %d", v.String(), p.curToken.Line), errType: InvalidAssignmentError}
```

Comment:
@hachi8833 The error message is different than Ruby's, but I think it's more understandable?